recipe/meta.yaml

```diff
@@ -14,17 +14,18 @@ source:
 build:
   number: 0
   noarch: python
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
     - setuptools
+    - pip
 
   run:
     - python
     - google-cloud-core >=0.28.0,<0.29dev
-    - google-api-core >=0.1.1,<0.2.0dev
+    - google-api-core >=1.0.0,<2.0.0dev
     - google-auth >=1.0.0
     - google-resumable-media >=0.2.1
     - requests >=2.18.0
```

Comment:
I'd like to propose that we remove the following dependencies because they don't appear in https://github.com/GoogleCloudPlatform/google-cloud-python/archive/bigquery-0.31.0.zip :
`google-auth >=1.0.0`
`requests >=2.18.0`

Reply:
Sounds good to me.